File: app/src/main/java/de/metzgore/beansplan/util/di/components/WeeklyScheduleFragmentSubComponent.java
package de.metzgore.beansplan.util.di.components;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import de.metzgore.beansplan.weeklyschedule.WeeklyScheduleFragment;

@Subcomponent
public interface WeeklyScheduleFragmentSubComponent extends AndroidInjector<WeeklyScheduleFragment> {
    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<WeeklyScheduleFragment> {
    }
}

File: app/src/main/java/de/metzgore/beansplan/util/di/components/WeeklyScheduleFragmentSubComponent.kt
package de.metzgore.beansplan.util.di.components

import dagger.Subcomponent
import dagger.android.AndroidInjector
import de.metzgore.beansplan.weeklyschedule.WeeklyScheduleFragment

@Subcomponent
interface WeeklyScheduleFragmentSubComponent : AndroidInjector<WeeklyScheduleFragment> {
    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<WeeklyScheduleFragment>()
}

